src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -293,129 +294,141 @@ export class BrowsingContextImpl {
 
   onTargetInfoChanged(params: Protocol.Target.TargetInfoChangedEvent) {
     this.#url = params.targetInfo.url;
+
+    if (this.#navigationStarted) {
+      this.#eventManager.registerEvent(
+        {
+          method: BrowsingContext.EventNames.NavigationStarted,
+          params: {
+            context: this.id,
+            navigation: null,
+            timestamp: BrowsingContextImpl.getTimestamp(),
+            url: this.#url,
+          },
+        },
+        this.id
+      );
+      this.#navigationStarted = false;
+    }
   }
 
   #initListeners() {
-    this.#cdpTarget.cdpClient.on(
-      'Page.frameNavigated',
-      (params: Protocol.Page.FrameNavigatedEvent) => {
```

Comment:
Even if the types are removed by auto formatting tool, this adds unnecessary noise to the PR, which can mask other changes